sdk/storage/storage-queue/README.md

```diff
@@ -93,10 +85,15 @@ For example, you can create following CORS settings for debugging. But please cu
 
 ## Examples
 
-### Import types
+### Import the package
 
-You can use the `const Azure = require("@azure/storage-queue");` shown above then use types and functions from `Azure`.
-Or you can selectively import certain types,
+To use the clients, import the package into your file:
+
+```javascript
+const AzureStorageQueue = require("@azure/storage-queue");
```

Comment:
How about the es6 import? 

Require the whole package is not friendly for bundlers.

Reply:
Discussed with Ramya and we're going to take this into a separate issue as part of browser-and-bundling-related work.